From 94da5567a7962a25fdc268ed284c15d20637c348 Mon Sep 17 00:00:00 2001
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Thu, 8 Feb 2018 17:50:23 +0100
Subject: [PATCH 09/27] vnc: zap dead code

RH-Author: Daniel P. Berrange <berrange@redhat.com>
Message-id: <20180208175041.5634-10-berrange@redhat.com>
Patchwork-id: 78940
O-Subject: [RHEL-7.5 qemu-kvm PATCH v1 09/27] vnc: zap dead code
Bugzilla: 1527405
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

From: Gerd Hoffmann <kraxel@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Reviewed-by: Peter Lieven <pl@kamp.de>
Reviewed-by: Daniel P. Berrange <berrange@redhat.com>
Message-id: 1446203414-4013-11-git-send-email-kraxel@redhat.com
(cherry picked from commit e081aae5ae01f5ff695ba9fee4e622053d8e4bfe)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 ui/vnc.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/ui/vnc.c b/ui/vnc.c
index ebb6484..9923d24 100644
--- a/ui/vnc.c
+++ b/ui/vnc.c
@@ -621,10 +621,6 @@ static void vnc_dpy_switch(DisplayChangeListener *dcl,
                                           width, height, NULL, 0);
 
     /* guest surface */
-#if 0 /* FIXME */
-    if (ds_get_bytes_per_pixel(ds) != vd->guest.ds->pf.bytes_per_pixel)
-        console_color_init(ds);
-#endif
     qemu_pixman_image_unref(vd->guest.fb);
     vd->guest.fb = pixman_image_ref(surface->image);
     vd->guest.format = surface->format;
-- 
1.8.3.1

